### PR TITLE
Split external process instrumentation tests again

### PR DESCRIPTION
**Pull request by mlopatkin**

### CODE REVIEW

**Comment by mlopatkin** [comment]
@bot-gradle test this

File: subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/inputs/process/instrument/AbstractProcessInstrumentationInDynamicGroovyIntegrationTest.groovy [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        }
+
+        where:
+        [varInitializer, processCreator, expectedPwdSuffix, expectedEnvVar, enableIndy] << testCases
+
+        title = processCreator.replace("command", varInitializer.description)
+        indyStatus = enableIndy ? "with indy" : "without indy"
+    }
+
+
```
Comment (bamboo) [inline subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/inputs/process/instrument/AbstractProcessInstrumentationInDynamicGroovyIntegrationTest.groovy:80]: 💄 I'm not sure the additional empty line adds any value here.

**Review by bamboo — APPROVED** [review]
Very nice! 👍

**Review by blindpirate — APPROVED** [review]
LGTM, thanks for making these changes!

**Comment by mlopatkin** [comment]
@bot-gradle test and merge

**Pull request merged**